plugins/common-aliases/common-aliases.plugin.zsh

```diff
@@ -34,7 +34,9 @@ alias -g P="2>&1| pygmentize -l pytb"
 
 alias dud='du -d 1 -h'
 alias duf='du -sh *'
-alias fd='find . -type d -name'
+if ! which fd >/dev/null 2>%1; then
+  alias fd='find . -type d -name'
+fi
```

Comment:
Syntax error, it should be `&1`. That said, it's more idiomatic to use `(( $+commands[fd] ))`:
```suggestion
(( $+commands[fd] )) || alias fd='find . -type d -name'
```

Reply:
Thanks!!